apps/frontend/web/src/components/car/CarModal.tsx

```diff
@@ -0,0 +1,440 @@
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalCloseButton,
+  FormControl,
+  FormLabel,
+  Input,
+  Select,
+  Button,
+  Heading,
+  Box,
+  Image,
+  FormErrorMessage,
+  useToast,
+} from '@chakra-ui/react';
+import { useForm } from 'react-hook-form';
+import { ImBin } from 'react-icons/im';
+import { RxUpdate } from 'react-icons/rx';
+import { AiOutlinePlusCircle } from 'react-icons/ai';
+import { Car as CarEntity } from '@apps/backend-api';
+import { Dispatch, SetStateAction, useState } from 'react';
+import DeleteDialog from './DeleteDialog';
+
+const carTypes = [
+  { label: 'Sedan', value: 'Sedan' },
+  { label: 'Suv', value: 'Suv' },
+  { label: 'Convertible', value: 'Convertible' },
+  { label: 'Minivan', value: 'Minivan' },
+  { label: 'Truck', value: 'Truck' },
+];
+
+const min = new Date().getFullYear() - 30;
+const max = new Date().getFullYear();
+const years = Array.from(new Array(max - min), (_, index) => max - index);
+
+const CarModal = ({
+  isOpen,
+  onClose,
+  isEdit,
+  car,
+  setCars,
+}: {
+  isOpen: boolean;
+  onClose: () => void;
+  isEdit: boolean;
+  car: CarEntity;
+  setCars: Dispatch<SetStateAction<CarEntity[]>>;
+}) => {
+  const {
+    register,
+    handleSubmit,
+    reset,
+    getValues,
+    formState: { errors },
+  } = useForm({
+    mode: 'onBlur',
+  });
+
+  const [isDeleteButtonPressed, setIsDeleteButtonPressed] = useState(false);
+  const toast = useToast();
+
+  const deleteCar = () => {
+    setIsDeleteButtonPressed(true);
+  };
+
+  const onSubmit = async (data: Partial<CarEntity>) => {
+    try {
+      const carData = { ...data };
+      delete carData.photo;
```

Comment:
```suggestion
  const onSubmit = async ({ photo, ...data }: Partial<CarEntity>) => {
    try {
      const carData = { ...data };
```
data ne contient plus photo